qiskit_ionq/ionq_provider.py

```diff
@@ -28,13 +28,20 @@
 
 import logging
 import os
+from dotenv import load_dotenv
 
 from qiskit.providers.exceptions import QiskitBackendNotFoundError
 from qiskit.providers.providerutils import filter_backends
 
 from . import ionq_backend
 
 logger = logging.getLogger(__name__)
+load_dotenv(
+    dotenv_path=os.path.join(
+        os.getcwd(), ".env"
+    ),  # Load .env file from current directory
+    override=True,  # Set override=True to allow for .env file to override env vars
+)
```

Comment:
It's pretty strange for an SDK to load env files like this on import. Why would you want this?

Reply:
In this mode, basically, all packages referenced before qiskti-ionq will have the normal env vars, then when ionq loads, it would potentially change these.

Could we use dotenv_values to avoid mucking with the environment?

Reply:
yeah that's a lot better thanks for the review!